beta/project-griffon/using-project-griffon.md

```diff
@@ -13,55 +13,51 @@ Project Griffon is in beta. Use of this beta product requires acceptance of term
 If you do not know your Adobe ID credentials, contact your Adobe administrator or see [how to log in](https://docs.adobe.com/content/help/en/core-services/interface/manage-users-and-products/getting-started-experience-cloud.html).
 {% endhint %}
 
-![](../../.gitbook/assets/image-2.png)
+3. In the beta access page, click **Register**. 
 
-If you see the following screen, for beta access, click **Register**.
+## Project Griffon sessions
 
-![](../../.gitbook/assets/image-7.png)
+To begin a session, you need to create a session and connect your device.
 
-## Begin a Project Griffon session
+### 1.Create a session
 
-### Create a session
+1. Log in to Project Griffon. 
 
-1. To begin, click the **Create Session** button on the upper right-hand corner of the screen.
+2. Click **Create Session** in the top right. 
 
-![](../../.gitbook/assets/image.png)
+3. Click **Start**.
 
-1. Click **Start**.
+4. In **Session Name**, enter a name to identify the session.  
+    This definition is used to launch your app from a URL and initiate the Project Griffon session. An example value might look like: `grifflab://default`.
+5. In the **Base URL** field, type your app's base deep link definition.
+6. Click **Next**.
 
-![](../../.gitbook/assets/image-4.png)
 
-1. In **Session Name**, enter a name to identify the session.  This definition is used to launch your app from a URL and initiate the Project Griffon session. An example value might look like: `grifflab://default`.
-2. In the **Base URL** field, type your app's base deep link definition.
-3. Click **Next**.
-
-![](../../.gitbook/assets/image-1.png)
-
-### Connect your device
+### 2.Connect your device
 
 1. Ensure that you see a screen that shows you a link, a QR code, and a PIN. 
```

Comment:
If it's not too much of a pain, can we change the step numbers to letters? (Instead of "1.", change it to "a."